integration/README.md

```diff
@@ -0,0 +1,60 @@
+# Integration Tests
+
+This tests core use cases for webboot such as:
+
+- retrieving and kexec'ing a Linux kernel,
+- uinit (user init), and
+- running unit tests requiring root privileges.
+
+## Usage
+
+Run the tests with:
+
+    go test
+
+     or 
+
+    UROOT_QEMU=/usr/bin/qemu-system-x86_64 UROOT_KERNEL=$HOME/linux/arch/x86_64/boot/bzImage go test
```

Comment:
did you test to see if it needed the full path? You should do that.
Also indicate this is an example, i.e. put 
for example:
on the line above.

Reply:
Further, you have this example command here. Why would you tell them to use RUNLOCAL if the command line you just showed them here will work?

Reply:
Since there are multiple options to run integration testing, I provided a specific distinction in which one they can use

Reply:
In these distinction, the user was able to pick which way to use. However, I realize not everyone uses their bashrc and edit their bashrc to keep variable names in there.